core/local/change.js

```diff
@@ -129,13 +90,66 @@ function isChildAdd (a /*: LocalChange */, b /*: LocalChange */) { return childO
 // $FlowFixMe
 function toString (a /*: LocalChange */) /*: string */ { return '(' + a.type + ': ' + (a.old && a.old.path) + '-->' + a.path + ')' }
 
+function identify (e /*: LocalEvent */, sameInodeChange /*: ?LocalChange */, samePathChange /*: ?LocalChange */) {
+  switch (e.type) {
+    case 'add':
+      return (
+        fileMoveFromTo(sameInodeChange, e) ||
+        fileMoveSuccessiveTo(sameInodeChange, e) ||
+        fileMoveIdenticalOffline(e) ||
+        fromEvent(e)
+      )
+    case 'addDir':
+      return (
+        dirMoveSuccessiveTo(sameInodeChange, e) ||
+        dirMoveFromTo(sameInodeChange, e) ||
```

Comment:
Thanks to the new renamed functions, we can easily see differences between branches...
E.g. why are those two in reverse order compared to the (file) `'add'` branch?

Reply:
Maybe some of those differences are normal, in which case we could add a comment explaining why some case is relevant on one branch but not on the other.

Reply:
Also, in case your editor has an outline view, you'll appreciate the new names natural sorting :smile: